src/mito2/src/read/scan_region.rs

```diff
@@ -582,7 +564,7 @@ impl ScanInput {
             predicate: None,
             memtables: Vec::new(),
             files: Vec::new(),
-            cache_manager: None,
+            cache_strategy: CacheStrategy::EnableAll(CacheManagerRef::default()),
```

Comment:
Not sure if `CacheStrategy::EnableAll(CacheManagerRef::default())` makes any sense when `CacheManagerRef::default()` does not contain any cache?

Reply:
fixed in 58b83032f111191c5ec17abcff4c2aef2dc1ec29